src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -76,11 +78,31 @@
     <Reference Include="System.Threading" />
     <Reference Include="System.Threading.Channels" />
   </ItemGroup>
+
+  <PropertyGroup>
+    <BinPlaceNative>true</BinPlaceNative>
+    <BinPlaceRuntime>false</BinPlaceRuntime>
```

Comment:
This will cause the assembly produced by this project `System.Net.Quic.dll` to not be binplaced into the runtime pack or the shared framework.

I'm not yet sure how to fix this without getting rid of binplacing entirely. Let me think about this a bit more.